package-scripts.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const path = require('path');
+
+const MOCHA = `node ${path.join('bin', 'mocha')}`;
+const TM_BUNDLE = 'JavaScript\\ mocha.tmbundle';
+
+function test (testName, params) {
+  const prefix = process.env.COVERAGE ? `nyc --no-clean --report-dir coverage/reports/${testName}` : '';
+  return `${prefix} ${MOCHA} ${params}`;
+}
+
+module.exports = {
+  scripts: {
+    build: `browserify ./browser-entry --plugin ./scripts/dedefine --ignore 'fs' --ignore 'glob' --ignore 'path' --ignore 'supports-color' > mocha.js`,
+    lint: {
+      default: 'nps lint.code lint.markdown',
+      code: 'eslint . bin/*',
```

Comment:
even though we're not running this on AppVeyor, it should still work on windows.  please use double quotes around globs: `'eslint . "bin/*"'`

Reply:
Gotcha. If anyone has access to a Windows machine too it'd be awesome to do a dry run and be sure these commands work as expected.